src/views/language/LanguageVitalityMeter.tsx

```diff
@@ -2,59 +2,33 @@ import React from 'react';
 
 import Deemphasized from '../../generic/Deemphasized';
 import Hoverable from '../../generic/Hoverable';
+import { LanguageData } from '../../types/LanguageTypes';
+
+import { computeVitalityMetascore } from './LanguageVitalityComputation';
 
 interface Props {
-  value?: string;
+  lang: LanguageData;
 }
 
-const LanguageVitalityMeter: React.FC<Props> = ({ value }) => {
-  if (!value) {
+const LanguageVitalityMeter: React.FC<Props> = ({ lang }) => {
+  const metascore = computeVitalityMetascore(lang);
+
+  if (metascore.score === null) {
     return <Deemphasized>Data not available</Deemphasized>;
   }
 
-  // Map Ethnologue vitality levels to meter values (inverted scale)
-  // Ethnologue scale: 1=National, 2=Regional, 3=Trade, 4=Educational, 5=Written, 6=Threatened, 7=Shifting, 8=Moribund, 9=Nearly Extinct, 10=Extinct
-  // We want: 9=National, 8=Regional, 7=Trade, 6=Educational, 5=Written, 4=Threatened, 3=Shifting, 2=Moribund, 1=Nearly Extinct, 0=Extinct
-  const getMeterValue = (vitality: string): number => {
-    switch (vitality.toLowerCase()) {
-      case 'national':
-        return 9;
-      case 'regional':
-        return 8;
-      case 'trade':
-        return 7;
-      case 'educational':
-        return 6;
-      case 'written':
-        return 5;
-      case 'threatened':
-        return 4;
-      case 'shifting':
-        return 3;
-      case 'moribund':
-        return 2;
-      case 'nearly extinct':
-        return 1;
-      case 'extinct':
-        return 0;
-      default:
-        // For any unknown values, return a neutral value
-        return 5;
-    }
-  };
-
-  const meterValue = getMeterValue(value);
-
   return (
-    <Hoverable hoverContent={value}>
+    <Hoverable
+      hoverContent={`Vitality Metascore: ${metascore.score.toFixed(1)} (${metascore.explanation})`}
+    >
       <meter
         min={0} // Extinct
-        low={3} // Shifting
-        high={7} // Trade
-        optimum={8} // Regional
-        max={9} // National
-        value={meterValue}
-        title={`Vitality: ${value}`}
+        low={3} // Shifting/Endangered
+        high={7} // Trade/Stable
+        optimum={8} // Regional/Institutional
+        max={9} // National/Living
```

Comment:
I love the comments, just fixing an error and explaining the color better
```suggestion
        low={3} // Shifting/Endangered -- below this, the meter is colored red
        high={7} // Trade/Stable -- below this, the meter is colored yellow
        optimum={8} // Regional -- tells the renderer that high and above is green
        max={9} // National/Institutional/Living
```